scripts/schemaGenerator.py

```
@@ -487,9 +517,16 @@ def generate_schema_for_dynamic_custom_form_allOf(
                 field["preRequisites"]["fields"], preRequisites, True
             ):
-                thenObj["properties"][field[schema_field_name]] = uiTypetoSchemaFn.get(
-                    field["type"]
-                )(field, dbConfig, schema_field_name)
+                fn = uiTypetoSchemaFn.get(field["type"])
+                if fn is not None:
+                    thenObj["properties"][field[schema_field_name]] = fn(
+                        field, dbConfig, schema_field_name
+                    )
+                else:
+                    warnings.warn(
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)